### PR TITLE
koord-scheduler: refactor framework extender init function

### PR DESCRIPTION
**Pull request by eahydra**
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: cmd/koord-scheduler/app/config/config.go [hunk 1]
```diff
@@ -19,6 +19,8 @@ package config
 import (
 	schedulerappconfig "k8s.io/kubernetes/cmd/kube-scheduler/app/config"
 
+	nrpinformers "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/informers/externalversions"
```
Comment (jasonliu747) [inline cmd/koord-scheduler/app/config/config.go:22]: `nrp`->`nrt`

Comment (eahydra) [reply cmd/koord-scheduler/app/config/config.go]: fixed.

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Comment by jasonliu747** [comment]
Noted: we should use koordlet to collect and report machine topology information in the later version. For further details, please check #232.

**Pull request merged**